reactor-core/src/main/java/reactor/core/scheduler/Scheduler.java

```diff
@@ -85,10 +95,15 @@ default Disposable schedule(Runnable task, long delay, TimeUnit unit) {
 	 * @return the {@link Disposable} that let's one cancel this particular delayed task,
 	 * or throw a {@link RejectedExecutionException} if the Scheduler is not capable of scheduling periodically.
 	 */
+	@Deprecated
```

Comment:
please add a `@deprecated` entry to the javadoc explaining the deprecation and alternative